### ISSUE TITLE
will pymtl be ported to Python3

### ISSUE THREAD

**Issue by CFAndy** [cornell-brg/pymtl#167]

**Comment by cbatten** [comment]
yes eventually! But our highest priority right now is working on the new version of PyMTL which will be faster and even more productive :)

**Comment by CFAndy** [comment]
Hi Batten
  You mean Mamba? That's awfully cool.  
  My ESL modeling tool is based on Python3. Looking forward to a Python3 release.
  Do you happen to have an estimated date?
-Andy

**Comment by cbatten** [comment]
This is probably low priority. We are working on the new version of PyMTL and getting all of that fleshed out and working in Python2. I guess we basically have a year to also do the Python3 port right: https://pythonclock.org/ :)
